src/store/modules/GlobalStore.js

```diff
@@ -1,10 +1,35 @@
 import api from '../api';
 
+const HOST_STATE = {
+  on: 'xyz.openbmc_project.State.Host.HostState.Running',
+  off: 'xyz.openbmc_project.State.Host.HostState.Off',
+  error: 'xyz.openbmc_project.State.Host.HostState.Quiesced',
+  transition: {
+    on: 'xyz.openbmc_project.State.Host.Transition.On',
+    reboot: 'xyz.openbmc_project.State.Host.Transition.Reboot'
+  }
+};
+
+const hostStateMapper = hostState => {
+  switch (hostState) {
+    case HOST_STATE.on:
+      return 'Running';
```

Comment:
Why "Running"? but then we store it as "on"?

Reply:
Yeah, was thinking to change this. I was originally following the copy we have existing in our angular app, but with the new design proposals, it looks like the status will be indicated with the icon only and no text